### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AznFreak**

### CODE REVIEW

File: tasks/configure.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+- name: Defines which parameters/files are editable by owners
```
Comment (SlothOfAnarchy) [inline tasks/configure.yml:2]: The task names specify what is being done in a task, like `Template edit_list configuration`.
The same thing with the other task names.

File: tasks/configure.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  template:
+    src: edit_list.conf.j2
+    dest: /home/sympa/etc/
+    owner: root
+    group: root
+    mode: 0644
+
+- name: Sympa main configuration file
+  template:
+	src: sympa.conf.j2
```
Comment (SlothOfAnarchy) [inline tasks/configure.yml:12]: Please indent using spaces.

File: tasks/configure.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+- name: Specifies how Sympa detects web crawler
+  template:
+  src: crawlers_detection.conf.j2
+  dest: /usr/share/sympa/default/
+  owner: root
+  group: root
+  mode: 0644
+
+- name: Virtual Host definition for Sympairtual Host definition for Sympa
```
Comment (SlothOfAnarchy) [inline tasks/configure.yml:34]: You pasted the name twice :D

File: tasks/configure.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  template:
+  src: robot.conf.j2
+  dest: /usr/share/doc/sympa/examples/sample/
```
Comment (SlothOfAnarchy) [inline tasks/configure.yml:37]: Is this file really used in a sample folder?

File: tasks/install.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    name: sympa 
+  when: ansible_pkg_mgr == "apt
+
+- name: create /data/project/sympa
+  file:
+    path: /data/project/sympa
+    state: directory
+    owner: sympa
+    group: sympa
+  notify: restart sympa
```
Comment (SlothOfAnarchy) [inline tasks/install.yml:13]: where is the handler?

File: tasks/main.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+---
+- include: install.yml
```
Comment (SlothOfAnarchy) [inline tasks/main.yaml:2]: `include` is deprecated. Use `include_tasks` or `import_tasks`

File: tasks/install.yml [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+- name: Install required apt packages
+  apt:
+    name: sympa 
+  when: ansible_pkg_mgr == "apt
+
+- name: create /data/project/sympa
+  file:
+    path: /data/project/sympa
+    state: directory
+    owner: sympa
```
Comment (SlothOfAnarchy) [inline tasks/install.yml:11]: Is the sympa user automatically created?

File: tasks/configure.yml [hunk 8]
```diff
@@ -0,0 +1,51 @@
+---
+- name: Defines which parameters/files are editable by owners
+  template:
+    src: edit_list.conf.j2
```
Comment (SlothOfAnarchy) [inline tasks/configure.yml:4]: Where are the templates?

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/configure.yml (hunk 2 above)
Comment (dasJ) [reply tasks/configure.yml:12]: *2 spaces

File: tasks/install.yml (hunk 5 above)
Comment (dasJ) [reply tasks/install.yml:13]: And pleas capitalize the handler

**Comment by dasJ** [comment]
Automatically bumping this issue because it was inactive for 46 days.

**Pull request closed without merging**